lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -593,9 +584,9 @@ def mock_run(args, **kwargs):
         assert tarfile.exists()
         assert md5file.exists()
 
-        todo_state = archive / "TODO" / tarfile.name
-        assert todo_state.is_symlink()
-        assert todo_state.samefile(tarfile)
+        # todo_state = archive / "TODO" / tarfile.name
+        # assert todo_state.is_symlink()
+        # assert todo_state.samefile(tarfile)
```

Comment:
Should this code be removed, now?  (If not, we should probably have some explanatory text indicating what this comment signifies.)